detection-rules/link_microsoft_low_reputation.yml

```diff
@@ -110,18 +109,18 @@ source: |
     or any(attachments,
            .file_type in $file_types_images
            and any(file.explode(.),
-                   any(ml.nlu_classifier(.scan.ocr.raw).intents, .name == "cred_theft")
+                   any(ml.nlu_classifier(.scan.ocr.raw).intents, .name == "cred_theft" and .confidence in~ ("medium", "high"))
            )
     )
     or (
       any(ml.nlu_classifier(body.html.inner_text).entities, .name == "urgency")
-      and not any(ml.nlu_classifier(body.current_thread.text).intents,
-                  .name == "benign" and .confidence == "high"
+      and any(ml.nlu_classifier(body.current_thread.text).intents, .name not in~ ("benign", "unknown")
```

Comment:
@aidenmitchell does this need to be an `all` to match our previous behavior? as written, we could get a `benign:high` and `cred_theft:low` and still flag here

Reply:
I don't think we could get a `benign` _and_ `cred_theft` intent, because we're only evaluating one thing, `body.current_thread.text`?

Reply:
`.intents` is an array because i think we can return multiple with different confidence levels, but @bfilar can confirm

Reply:
so if the model returns `benign` that would be the only intent we return. If `benign` is not present in intent array would could return something like: `(cred_theft,steal_pii)`